src/blog/2024-10-14/index.md

```diff
@@ -0,0 +1,45 @@
+title: Neotokyo;Rebuild Development Progress Update
+summary: Update on project progress over the last couple of months
+author: kinoko and blaberry 
+
+# Introduction
+2024-10-14 kinoko and blaberry
+
+Hi, it has been quite a few months since the last dev progress update. Better late than never though, so let’s go over what has happened and where the project is heading!
+
+# What’s the state of NT;RE?
+
+The state of rebuild is looking great. Most of the parity with the current game has been achieved, with some things still needing to be ironed out, and now the devs have started focusing some attention on the post-parity/QoL stuff. To name a few examples - we’re integrating plugin and competitive mode as native server features, clients have custom crosshair support, there's a new menu UI and we’re looking at implementing native gamemodes previously available as sourcemods. Since there has been a lot added over the months, head over to our website for full patch notes: [https://neotokyorebuild.github.io/](https://neotokyorebuild.github.io/)
```

Comment:
The reader is already in the website at this point, so no need for the last section or just mention the blog.

Reply:
True true, makes sense

Reply:
done